.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,63 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+type: Bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    attributes:
+      label: OS
+      description: |
+        Describe your Operating System. Example:
+          - **OS**: Ubuntu 18.04
```

```suggestion
        Describe your Operating System. Examples: Ubuntu 18.04, CentOS 8 Stream
```
No need to repeat "OS:" in example text